modules/virt-using-wasp-agent-to-configure-higher-vm-workload-density.adoc

```diff
@@ -46,13 +130,23 @@ $ oc create clusterrolebinding wasp --clusterrole=cluster-admin --serviceaccount
 ----
 $ oc adm policy add-scc-to-user -n wasp privileged -z wasp
 ----
+
+. Wait for the worker nodes to sync with the new configuration by running the following command:
```

Comment:
See my previous comment about whether the goal of the step is to wait or to sync.